tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/Makefile.json

```diff
@@ -1,11 +1,6 @@
 {
   "ENTRY": "IotHttpsClient_AddHeader",
 
-  # A CBMC pointer is an object id followed by an offset into the object.
-  # The size of the offset is limited by the size of the object id.
-  "CBMC_OBJECTID_BITS": "7",
-  "CBMC_MAX_OBJECT_SIZE": "\"((UINT32_MAX>>CBMC_OBJECTID_BITS)>>1)\"",
-
   "CBMCFLAGS":
```

Comment:
Why deleting these definitions, where are the definitions you are using?

Reply:
I added `#included "cbmc.h"` at the top of `global_state_HTTP.c`. These definitions now come from "cbmc.h".